src/main/java/io/appium/java_client/AppiumDriver.java

```diff
@@ -223,6 +223,15 @@ protected static Capabilities substituteMobilePlatform(Capabilities originalCapa
         return contextName;
     }
 
+    public AppiumServerStatus getStatus() {
```

Comment:
I'd rather simply return a map out of it. The status output is not constant and differs for different drivers

Reply:
This is related to Appium Server status that shows the build version info when we call "http://localhost:4723/wd/hub/status" while running appium locally. The idea was to add the ability in the driver to return its build version when someone calls getStatus() method which returns AppiumServerStatus object having getVersion() method to return the build version value. 
By returning the map, we might be forcing the user to know the key (i.e. "version" in this case) and pass this to get the value from the map.
Am I missing anything here?

Reply:
> By returning the map, we might be forcing the user to know the key (i.e. "version" in this case) and pass this to get the value from the map.

I'm fine with that

Reply:
@mykola-mokhnach, @jlipps , @saikrishna321, @SrinivasanTarget 
My idea was to keep it simple and scalable by having a separate Class for managing the status (and to have ease in adding new fields, if any in the future) rather than expecting users to know about the internal fields. 
But if you guys ok with returning the versionMap from getStatus() method implemented here, then I can go ahead and change the code to return versionMap info?

Reply:
I would go with just returning the versionMap and let the user pick the values from it.

Reply:
Updated the code as suggested and done formatting. CI tests are still breaking which needs to be fixed separately.